modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
+
+When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installation program to encrypt the root volumes for the nodes in the cluster. The installation program also encrypts the default storage class with this key. This allows you to encrypt storage volumes with your selected key, instead of the platform's default account key.
```

Comment:
Based on https://issues.redhat.com/browse/CORS-2563?focusedCommentId=21842953&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21842953, 
```
The installation program also encrypts the default storage class with this key
``` 
only happens when the custom key was specified in platform.{cloud_type}.defaultMachinePlatform field in the install-config.yaml. This should be highlighted in this section.